docs/source/basic/terms.rst

```diff
@@ -0,0 +1,70 @@
+Terms
+=====
+
+This page provides an overview of the terms used in ``alpaka`` and the relationships between them.
+
+Platform
+--------
+
+- A ``platform`` contains information about the system, e.g. the available devices. 
+- Depending on the platform, it also contains a runtime context.
+- A ``platform`` can be shared by many devices.
+
+Device
+------
+
+- A ``device`` represent a compute unit, such as a CPU or a GPU.
+- Each ``device`` is bounded to a specific ``platform``.
+- Each ``device`` can be used by many specific ``accelerators``.
+
+Work division
+-------------
+
+- Describes the domain decomposition of a contiguous N-dimensional index domain in ``blocks``, ``threads`` within a ``block``, and ``elements`` per ``thread``.
+- A ``work division`` has limitations depending on the ``kernel`` function and ``accelerator``.
+
+Accelerator
+-----------
+
+- Describes "how" a kernel work division is mapped to device threads.
```

Comment:
We give work division separately to the kernel, not use the workdiv from the acc?

Reply:
The user is giving the work division to the kernel exec call but at the end this is only the interface. I do not like describe the interface in words, my intention is to describe what different concepts do.
In this case the accelerator performs the mapping from indices to physical threads.